app/controllers/calculator_constants_controller.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+class CalculatorConstantsController < ApplicationController
+  def index
+    @calculator_constants = CalculatorConstant.all
+  end
+
+  def update
+    updated_fields = []
+    params.each do |key, value|
+      update_calculator_constant(key, value, updated_fields)
+    end
+    unless updated_fields.empty?
+      flash[:success] = {
+        updated_fields: updated_fields
+      }
+    end
+    redirect_to action: :index
+  end
+
+  private
+
+  def update_calculator_constant(name, value, updated_fields)
+    if CalculatorConstant.exists?(name: name)
+      constant_field = CalculatorConstant.where(name: name).first
```

Comment:
This is going to result in a lot of SQL calls. I think you can instead be more performant by a single query:

```ruby
CalculatorConstant.where(name: ["AVGDODBAH","IGMCKY",...])
```

Reply:
Good point. I made the change. Thans!